tools/validators/instance_validator/validate/handler.py

```diff
@@ -200,11 +207,11 @@ def _ParseArgs(self, args):
 
     parser.add_argument(
         '-r', '--report-filename',
-        dest = 'report_filename',
-        required = False,
-        default = None,
-        help = 'Filename for the validation report',
-        metavar = 'report-filename')
+        dest='report_filename',
```

Comment:
nit/question: Is param_name=param_value the correct style for Python named parameters? It looks wrong to me without whitespace but I can adjust if that's the convention.

Reply:
pylint was complaining so I changed it similar to the ones above

Reply:
I have this error from the linter:

validate/entity_instance.py:382:43: C0326: No space allowed around keyword argument assignment
  def IsValidEntityInstance(self, universe = None, entity_instances = None):
                                           ^ (bad-whitespace)
validate/entity_instance.py:382:68: C0326: No space allowed around keyword argument assignment
  def IsValidEntityInstance(self, universe = None, entity_instances = None):
                                                                    ^ (bad-whitespace)